service/job/api/src/main/java/org/eclipse/kapua/service/job/JobAttributes.java

```diff
@@ -23,4 +23,5 @@
 
     public static final String ENDED_ON = "endedOn";
     public static final String STARTED_ON = "startedOn";
+    public static final String DESCRIPTION = "description";
```

Comment:
This value is inherited from `KapuaNamedEntityAttributes`. 
Please remove.

Reply:
@Coduz  Value removed.